src/commands/batch.py

```diff
@@ -62,15 +63,15 @@ def list(limit):
             query = session.query(Batch) \
                            .order_by(Batch.created_date.desc()) \
                            .limit(limit)
-            headers = ['ID', 'Date', 'Name', 'Nodes']
+            headers = ['ID', 'Date', 'Name', 'Arguments', 'Nodes']
```

Comment:
The `Name` header should be changed to `Command` so it is consistent with the above table.